### DISCUSSION TITLE
Location tracker always away

### DISCUSSION THREAD

**Discussion by Sh1bbyz** [alandtse/tesla#403] [Q&A] [answered]
The location only shows "away" no matter what. Is there something that you have to do to get this working?
I have entered my home adress as "home" in the navigation.

**Comment by carleeno** [comment, marked as answer]
You need to set your home location in Home Assistant itself, and make sure the geofence radius is big enough to cover your garage/driveway.

Then when the car is within your HA's home geofence, the location tracker will show "home". It doesn't use the car's navigation information.

You can also set up other locations in HA such as work, etc. This will also allow the location tracker for you mobile apps to also work.

**Reply by Sh1bbyz** [reply]
Thank you very much! This did indeed work!
